src/stylesheets/components/_disclaimer.scss

```diff
@@ -1,10 +1,21 @@
 .usa-disclaimer {
   background-color: $color-gray-lightest;
+  padding-bottom: 7px;
```

Comment:
@msecret I think this was commented on an old version of the PR. The new work is in `_banner.scss` and this code doesn't exist.